compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -217,6 +224,7 @@ def __init__(
             _p = pathlib.Path(self.identity_mapping_config_path)
             if not _p.exists():
                 warnings.warn(f"Identity mapping config path not found ({_p})")
+        self.public = public is True
 
```

Comment:
Och, we don't seem to have a `tests/unit/test_config.py`.  This would be an easy unit test for that:

```python
@pytest.mark.parametrize("public", (None, True, False, "a", 1))
def test_public(public)
    c = Config(public=public)
    assert c.public == public is True
```